presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.cpp

```diff
@@ -863,6 +863,11 @@ connector::hive::LocationHandle::TableType toTableType(
       return connector::hive::LocationHandle::TableType::kNew;
     case protocol::TableType::EXISTING:
       return connector::hive::LocationHandle::TableType::kExisting;
+    // Temporary tables are written and read by the SPI in a single pipeline.
+    // So they can be treated as New. They do not require Append or Overwrite
+    // semantics as applicable for regular tables.
+    case protocol::TableType::TEMPORARY:
```

Comment:
Nit: This along with the comment can be moved to the first case

Reply:
Done.